dashboard/test/models/script_test.rb

```diff
@@ -1384,13 +1390,15 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
       ScriptDSL.parse(old_dsl, 'a filename')[0][:lesson_groups]
     )
     assert script.script_levels.first.challenge
+    original_script_level_id = script.script_levels.first.id
```

Comment:
You can see that this is an existing bug if you check out staging and run this test with this line + the assert below to the test.

Reply:
sorry... what behavior is a bug?

Reply:
just confirming, the bug is "ScriptLevel is destroyed and recreated when 'challenge' status changed" as mentioned in the PR description, and this PR does not currently fix it, correct?